libs/evm-tools/package.json

```diff
@@ -47,12 +47,13 @@
   "dependencies": {
     "@ledgerhq/cryptoassets": "workspace:^",
     "@ledgerhq/live-env": "workspace:^",
-    "@ledgerhq/live-network": "workspace:^",
+    "axios": "^1.6.5",
```

Comment:
I'm noticing that we use many different versions of axios, even across our own libs... Unsure of the potential risk/impact, we should probably align them all at some point.